operator/controllers/execution/scans/parse_reconciler.go

```diff
@@ -103,9 +103,13 @@ func (r *ScanReconciler) startParser(scan *executionv1.Scan) error {
 			BackoffLimit:            &backOffLimit,
 			Template: corev1.PodTemplateSpec{
 				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"app.kubernetes.io/managed-by=securecodebox": "securecodebox",
+					},
```

Comment:
Is this pattern correct? `managed-by=securecodebox: securecodebox` seems redundant.